src/components/select/multi-select/multi-select.spec.tsx

```diff
@@ -779,6 +783,122 @@ describe("MultiSelect", () => {
     });
   });
 
+  describe("when the filter text contains whitespace,", () => {
```

Comment:
Same comment as above wrt to tidying up the output when these tests run and maybe removing some nesting of describes